app/register/page.tsx

```diff
@@ -296,10 +313,11 @@ const Register: React.FC = () => {
             xs={12}
             sx={{
               color: "#11142D",
+              fontWeight: "bold",
             }}>
             <Typography align="center">
               {registerPageDetails.registerPageHaveAccountSection}{" "}
-              <a href="/login">
+              <a href="/login" style={{color:"#003C70", textDecoration:"none"}}>
```

Comment:
Use anchor tag alternative for MUI (Link)

Reply:
**done.**